app/src/main/java/in/ac/bits_pilani/goa/ard/utils/AHC.java

```diff
@@ -58,4 +62,14 @@
      * background image for navigation drawer.
      */
     public static final String FDR_NAV_DRAWER_IMAGE_LIST = "navDrawerImages";
+
+    /**
+     * Method to get pixel value corresponding to input dp.
+     * @param context of calling method.
+     * @param dp value to be converted in dp.
+     * @return converted value in pixels.
+     */
+    public static float dpToPx(final Context context, final float dp) {
```

Comment:
If this methods works for vector drawable, then I don't think we need to use pngs right?
Otherwise it is fine